io.github.torrent_file_editor.Torrent-file-editor.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: io.github.torrent_file_editor.torrent-file-editor
```

Comment:
this is case sensitive and doesn't match the filename or the patch.

Reply:
fixed